plugins/catalog-common/src/permissions.ts

```diff
@@ -24,6 +27,13 @@ import { createPermission } from '@backstage/plugin-permission-common';
  */
 export const RESOURCE_TYPE_CATALOG_ENTITY = 'catalog-entity';
 
+/**
+ * Convenience type for catalog entity
+ * {@link @backstage/plugin-permission-common#ResourcePermission}s.
+ * @public
```

Comment:
probably need to be updated to import from /alpha in the end to end test. We should look into this 👍

Reply:
Hmm - the import is actually happening in [catalog-react](https://github.com/backstage/backstage/pull/10066/files#diff-9e472129beb90ec10499f660b03f6baa4b389400f26992839539d4e4c2d63e7cL18), so I'm not sure whether the issue is limited to the e2e tests. Perhaps there is a wider issue with packages importing `@alpha` types from other packages inside the monorepo? Is this something we already have working examples for?

Reply:
Did a bit of digging - I suspect I might be missing a bunch of context on this topic, but at face value it seems to me that we're missing a solution which allows importing `@alpha` types across monorepo packages. Without that, `.d.ts` files in package output will be incorrect, because they'll attempt to import alpha things from `@backstage/plugin-x` that are only present in `@backstage/plugin-x/alpha`.

One possible avenue to address this might be using the [paths](https://www.typescriptlang.org/tsconfig#paths) option in TS to adjust the way imports ending in /alpha (and /beta) are resolved. That way the source code can use the right module path. Example of this in 54877613be60e58715c257e6e7affd6258ee8769 and fab2c6f027f628e42429b2a273884d922f94b4f8 for discussion (and just to get the build green).

Downside of this solution is that just like today, there's no local TS errors when you import something marked as `@alpha` or `@beta` _without_ the suffix.

Reply:
Pushed a32f399edd to work around this issue in this PR, perhaps we can continue the discussion in https://github.com/backstage/backstage/pull/10128.